packages/common/lib/types/OpenID4VCIVersions.types.ts

```diff
@@ -0,0 +1,10 @@
+export enum OpenId4VCIVersion {
+  VER_9 = 'version9',
+  VER_11 = 'version11',
+  VER_UNKNOWN = 'unknown version',
+}
+
+export enum URLSchemes {
```

Comment:
These are URI Schemes. Suggest to rename to DefaultURISchemes, as there always is the option to use your own scheme